games/xworld3d/maps/XWorldWalls.py

```diff
@@ -0,0 +1,34 @@
+from xworld_env import XWorldEnv
+import os
+import random
+
+class XWorldWalls(XWorldEnv):
+    def __init__(self, item_path):
+        super(XWorldWalls, self).__init__(
+            item_path=item_path,
+            max_height=7,
+            max_width=7)
+
+    def _configure(self):
+        self.set_dims(7, 7)
+        self.set_goal_subtrees(["animal", "fruit", "shape"])
```

Comment:
goal substrees are incorrect. If you didn't write this file, just delete it instead of purely copying from xworld2d

Reply:
deleted this file